### PR TITLE
Update tic_tac_toe.py

### PR DESCRIPTION
**Pull request by harshitashyale**
Made a minor fix in printBoard function

### CODE REVIEW

**Comment by githubparticipant** [comment]
seems like there's no score tracking logic yet?

**Comment by harshitashyale** [comment]
that's for the next release

File: tic_tac_toe.py [hunk 1]
```diff
@@ -2,7 +2,7 @@ def sum(a, b, c ):
     return a + b + c
 
 
-def printBoard(playerY, playerX):
+def printBoard(playerX, playerY):
```
Comment (githubparticipant) [inline tic_tac_toe.py:5]: switch variables around

**Comment by githubparticipant** [comment]
hello?

**Comment by githubparticipant** [comment]
make code simpler

File: tic_tac_toe.py (hunk 1 above)
Comment (githubparticipant) [reply tic_tac_toe.py:5]: - [ ] my reply

Comment (githubparticipant) [inline tic_tac_toe.py:5]: switch variables around

Comment (githubparticipant) [reply tic_tac_toe.py:5]: asdf

Comment (githubparticipant) [reply tic_tac_toe.py:5]: recheck the logic

Comment (githubparticipant) [reply tic_tac_toe.py:5]: recheck the logic

File: tic_tac_toe.py [hunk 2]
```diff
@@ -2,7 +2,7 @@ def sum(a, b, c ):
...
     zero = 'X' if playerX[0] else ('O' if playerY[0] else 0)
     one = 'X' if playerX[1] else ('O' if playerY[1] else 1)
     two = 'X' if playerX[2] else ('O' if playerY[2] else 2)
```
Comment (githubparticipant) [inline tic_tac_toe.py:8]:
```suggestion
    two = 'X' if playerX[2] else ('O' if playerY[2] else 2)- edited
```

**Pull request closed without merging**